lib/eosknowledgeprivate/ekn-resources.gresource.xml

```diff
@@ -2,5 +2,7 @@
 <gresources>
   <gresource prefix="/com/endlessm/knowledge-private">
     <file preprocess="xml-stripblanks">EknRuntimeDocumentViewer.ui</file>
+    <file preprocess="xml-stripblanks">EknMediaBin.ui</file>
```

Comment:
Which reminds me I started writing a compact binary formar for GMarkupParseContext and I never finish it.
But the main idea is to have a compact generic binary format that would allow us to avoid all the heavy part of parsing the xml in a template and do something like

```<file preprocess="xml-convert-binary">EknMediaBin.ui</file>```